jobs/aws/eks-anywhere/eks-anywhere-brew-update-postsubmits.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-anywhere:
+    - name: brew-update-postsubmit
+      always_run: false
+      run_if_changed: "release/triggers/brew-version-release/*"
+      branches:
+        - ^release-.*$
+      max_concurrency: 10
+      cluster: "prow-postsubmits-cluster"
+      skip_report: false
+      extra_refs:
+        - org: eks-anywhere-brew-pr-bot
+          repo: homebrew-tap
+          base_ref: main
+      decoration_config:
+        gcs_configuration:
+          bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
+          path_strategy: explicit
+        s3_credentials_secret: s3-credentials
+      spec:
+        serviceaccountName: postsubmits-build-account
+        automountServiceAccountToken: false
+        containers:
+          - name: build-container
+            image: public.ecr.aws/eks-distro-build-tooling/builder-base:8c4782ea5b009063433b37f68172efb74ee70c6b.2
+            command:
+              - bash
+              - -c
+              - >
+                trap 'touch /status/done' EXIT
```

Comment:
We don't need this, this is for jobs that have the buildkit sidecar container, to stop the sidecar container